pkg/client/run.go

```diff
@@ -35,6 +39,15 @@ import (
 
 // Run runs the event loop processing with given handler
 func Run(conf *config.Config) {
+	metricsPort := ":2112"
```

Comment:
I think a better name for this would be `listenAddress` or something
I think the format is `"host:port"`

Would it be better allowing to override it with an environment variable?

Reply:
Yes, sure. I will add environment variable for that